test/common/common/log_verbosity_update_test.cc

```diff
@@ -232,7 +274,23 @@ TEST(FineGrainLog, inOrderUpdate) {
   EXPECT_EQ(p->level(), spdlog::level::warn);
 }
 
-TEST(FineGrainLog, invalidLevelUpdate) {
+TEST(FineGrainLog, earlyExitGlobUpdate) {
+  const std::string file_1 = "envoy/src/foo.cc";
+  std::atomic<spdlog::logger*> flogger{nullptr};
+  getFineGrainLogContext().initFineGrainLogger(file_1, flogger);
+
+  const std::pair<absl::string_view, int> update_1 =
+      std::make_pair("f??", static_cast<int>(spdlog::level::warn));
+  const std::pair<absl::string_view, int> update_2 =
+      std::make_pair("ff?", static_cast<int>(spdlog::level::info));
+  getFineGrainLogContext().updateVerbositySetting({update_1, update_2});
+
+  SpdLoggerSharedPtr p = getFineGrainLogContext().getFineGrainLogEntry(file_1);
+  EXPECT_NE(p, nullptr);
```

Comment:
this (and likely some other places in this) should be an `ASSERT_NE` otherwise we'll go and execute the next line and segfault. See: http://google.github.io/googletest/primer.html#assertions

Reply:
Thank you! Done!